stable/database/files/nuosm

```diff
@@ -268,6 +268,16 @@ function perform_restore() {
 
     # restore data and/or fix the metadata
     log "restoring archive and/or clearing restored archive physical metadata"
+
+    if [ "$download_dir" != "$DB_DIR" ] ; then
+       # if not backup set then rename $download_dir to $DB_DIR.
+       if test -f "$download_dir/1.atm" ; then
+          rm -rf $DB_DIR
```

Comment:
This seems risky. Do we actually need to completely delete the contents of `DB_DIR`? What is the scenario here?

Reply:
If you look up a few lines,  you'll see that the directory being deleted was just created and is empty.